pdns/lua-recursor4.cc

```diff
@@ -343,6 +343,12 @@ RecursorLua4::RecursorLua4(const std::string& fname)
   d_lw->registerFunction("getRecords", &DNSQuestion::getRecords);
   d_lw->registerFunction("setRecords", &DNSQuestion::setRecords);
 
+  d_lw->writeFunction("addPolicyTag", [](vector<string>* tags, const string& tag) {
+      if (tags) {
+        tags->push_back(tag);
+      }
+    });
+
```

Comment:
Options I have in mind:
1. make table.insert work on it, would be just about as weird though
2. change from `return 5` to `return 5, {"foo", "ipsum"}`

I don't think 1 is important. 2 might be nice but what you have now is fine with me!